statchatexpress/app.js

```diff
@@ -0,0 +1,75 @@
+import express from 'express';
+import cors from 'cors';
+import morgan from 'morgan';
+import { sequelize } from './database.js';
+import { User, Stats } from './models/index.js';
+
+const app = express();
+
+app.use(cors())
+app.use(express.json()); // Middleware for parsing JSON bodies from HTTP requests
+app.use(morgan('combined'))
+
+// Route to get all users
+app.get('/users', async (req, res) => {
```

Comment:
Do you need this endpoint? What were you planning to use it for?

Reply:
yes, this is just to view all of the users when i type in localhost3000/users and it helps me see wa